src/components/split-button/split-button-children.style.js

```diff
@@ -20,7 +20,7 @@ const StyledSplitButtonChildrenContainer = styled.div`
       margin-top: 3px;
       margin-bottom: 3px;
       min-width: 100%;
-      text-align: left;
+      text-align: ${align === 'right' ? 'right' : 'left'};
```

Comment:
There's a defaultProp set for align, so I think this could just be set with `text-align: ${align}`

Reply:
Yup ,you're right :) Fixed